modules/tinymce/src/core/test/ts/browser/newline/InsertNewLineTest.ts

```diff
@@ -484,4 +484,214 @@ describe('browser.tinymce.core.newline.InsertNewLineTest', () => {
       });
     });
   });
+
+  context('newline_behavior "block"', () => {
+    before(() => {
+      hook.editor().options.set('newline_behavior', 'block');
+    });
+
+    after(() => {
+      hook.editor().options.unset('newline_behavior');
+    });
+
+    it('Split block in the middle', () => {
```

Comment:
Should these be prefixed by the ticket id?

Reply:
This is a tricky one I guess... because the behaviour itself is not directly related this. It probably wouldn't hurt to add it because it's related.

Reply:
I'm still not used to doing that. I'll add it to the `context` block.